.github/workflows/deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy
```

Comment:
Can we rename this and the filename to something clearer, i.e. Deploy to AWS, cd-aws.yml

Reply:
Done! https://github.com/ntampakas/tlsn/commit/e9776aa45d9adc35a597b4f24000bb08bb51ca79